docs/sources/set-up/migrate/from-static.md

```diff
@@ -51,6 +51,8 @@ This conversion allows you to take full advantage of the many additional feature
 
     * _`<INPUT_CONFIG_PATH>`_: The full path to the configuration file for Grafana Agent Static.
     * _`<OUTPUT_CONFIG_PATH>`_: The full path to output the {{< param "PRODUCT_NAME" >}} configuration.
+  
+1. Stop any already running Static Agent to avoid the port conflict.
```

Comment:
```suggestion
1. [Stop][] Grafana Agent static.
```

Lets keep the step simple, and link to the Grafana Static docs for teh stop commands.

Somewhere after line 366 in this topic, you will find a block of links. add the following link:

```
[Stop]: https://grafana.com/docs/agent/latest/static/set-up/start-agent/
```

Reply:
Sure, I will fix it.